webrtc.html

```diff
@@ -10139,6 +10141,23 @@ <h2>
             </section>
           </div>
         </section>
+        <div id="rtcencodeoptions"><!-- kept for candidate amendments management purposes --></div>
+        <section id="rtcencodeoptions">
+          <h3>
+            <dfn>RTCEncodeOptions</dfn> Dictionary
```

Comment:
Add an export?

Reply:
(not needed - as an IDL term, this gets exported automatically)